### ISSUE TITLE
Animation control

### ISSUE THREAD

**Issue by hellozebra73** [scanny/python-pptx#400]
Hi
I wonder if there are any plans to include animation manipulation features in python-pptx anytime soon?

Thanks

**Comment by imranal** [comment]
Yeah, it would be handy to have this function implemented (not sure if it is already). 

I am trying myself to generate equations with SymPy and save them in a ppt file, where I want to animate these equations. Animations are useful when you want to record a presentation as a video.

**Comment by wujunnan0929** [comment]
if we could add animation effect on a shape, that will be really convient so much. We indeed need this feature. Thanks.

**Comment by Arya-Programmer** [comment]
> Yeah, it would be handy to have this function implemented (not sure if it is already). 
> 
> I am trying myself to generate equations with SymPy and save them in a ppt file, where I want to animate these equations. Animations are useful when you want to record a presentation as a video.

I am working on a similar project, did you find a way to go about that? The only difference is I have to get the equations from a Word file.
